src/tpm2-crypto.c

```diff
@@ -667,7 +668,8 @@ static UsefulBuf cbor_pcr_assertions(UsefulBuf buf, uint32_t pcr_update_ctr,
             for (unsigned ii = 0; ii < sel->pcrSelections[i].sizeofSelect; ii++)
                 pcrs |= sel->pcrSelections[i].pcrSelect[ii] << 8*ii;
 
-            QCBOREncode_OpenMapInMap(&ctx, alg);
+            QCBOREncode_OpenMap(&ctx);
+                QCBOREncode_AddSZStringToMap(&ctx, "algo_name", alg);
```

Comment:
```suggestion
                QCBOREncode_AddUInt64ToMap(&ctx, "algo_id", sel->pcrSelections[i].hash);
```
After that change `alg` and `alg_name()` are no longer used and can be removed.

Reply:
See https://github.com/fobnail/docs/pull/18 for reasoning behind this change

Reply:
Done